app/src/main/java/com/technopark/hw_1/SecondFragment.java

```diff
@@ -0,0 +1,34 @@
+package com.technopark.hw_1;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.Fragment;
+
+public class SecondFragment extends Fragment {
+    private final NumberModel mNumberModel;
+
+    public SecondFragment(NumberModel numberModel) {
```

Comment:
**IMPORTANT**
It's really bad idea to put data through fragment constructor, or setter :)
Android system can recreate fragments by itself, so you're going to miss the data you've passed using constructor or setter

Please use `arguments` to pass data to fragments. Just google "android fragment arguments"